roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -3,6 +3,9 @@
 - set_fact:
     l_insecure_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(openshift.docker.insecure_registries)) }}"
   when: openshift.docker.insecure_registries
+- set_fact:
+    l_additional_crio_registries: "{{ '\"{}\"'.format('\", \"'.join(openshift.docker.additional_registries)) }}"
+  when: openshift.docker.additional_registries
```

Comment:
I can easily add it, but do we really want this?  Not going by default on `docker.io` for each image not found in the specified registries looks like a "feature" to me